modules/API/pages/gsql-endpoints.adoc

```diff
@@ -0,0 +1,6054 @@
+----
+--
+====
+
+'''
+
+== loading job
+
+=== get loading job names
+
+`GET /gsql/v1/loading-jobs`
+
+This endpoint is used to get all loading job names in a graph.
+
+==== Parameters:
+
+//[width="100%",cols="25%,25%,50%",options="header",]
+[%autowidth]
+|===
+| Name | Required | Description
+
+| `graph`
+| yes
+| the loading job names of which graph
+|===
+
+==== Example
+
+[,tabs]
+====
+Sample Request::
++
+--
+[source,bash]
+----
+curl -H 'Content-Type: application/json' -X GET 'http://localhost:14240/gsql/v1/loading-jobs?graph=g'
+----
+--
+Sample Response::
++
+--
+[source.wrap,console]
+----
+{"error":false,"message":"","jobNames":["incidents_fraud_report_company_csv"]}
+----
+--
+====
+
+'''
+
+=== create/update a loading job
+
+`POST /gsql/v1/loading-jobs`+
+`PUT /gsql/v1/loading-jobs`
+
+
+This endpoint is used to create a new loading job/update an existing loading job.
+
+==== Parameters:
+
+//[width="100%",cols="25%,25%,50%",options="header",]
+[%autowidth]
+|===
+| Name | Required | Description
+
+| `graph`
+| yes
+| create/update loading job of which graph
+|===
+
+==== Example
+
+[,tabs]
+====
+Sample Request::
++
+--
+[source,bash]
+----
+curl -H 'Content-Type: text/plain' -X POST/PUT 'http://localhost:14240/gsql/v1/loading-jobs?graph=g' -d '<loading job statement>'
+----
+--
+Sample Response::
++
+--
+[source.wrap,console]
+----
+{"error":false,"message":"Successfully created loading job: <jobName>"}
+----
+--
+====
+
+'''
+
+=== get loading job info
+
+`GET /gsql/v1/loading-jobs/{jobName}`
+
+This endpoint is used to get a specific loading job's information.
+
+==== Parameters:
+
+//[width="100%",cols="25%,25%,50%",options="header",]
+[%autowidth]
+|===
+| Name | Required | Description
+
+| `graph`
+| yes
+| get the loading job information under which graph
+
+| `verbose`
+| no
+| default: false, true means get loading job information in json format

```

Comment:
this description is not correct. Please describe clearly when `verbose=false`, what will return, and `verbose=true` what will return